inspire/dojson/hep/schemas/hep-0.0.1.json

```diff
@@ -593,6 +612,45 @@
                 }
             }
         },
+        "succeeding_entry": {
```

Comment:
What is this field used for? It seems like a relationship but it is not very clear where it points to. Could be useful a description like https://github.com/inspirehep/inspire-next/pull/400/files#diff-3db794b5dfd367ebcb5f09729f2490f5R543 ? And maybe use `curated_relation` in all relationships?